src/main/g8/project/Dependencies.scala

```diff
@@ -0,0 +1,8 @@
+import sbt._
+
+object Dependencies {
+  lazy val akkaHttpVersion = "$akka_http_version$"
+  lazy val akkaHttp        = "com.typesafe.akka" %% "akka-http" % akkaHttpVersion
```

Comment:
might be useful to have the akka version as parameter and then explicitly set the `akka-stream` dependency here, because I think that if you want to use akka-http with akka 2.5.x you have to define the stream dependency?